src/components/table/table.stories.js

```diff
@@ -113,96 +113,52 @@ const setCurrentPage = ({ totalRecords, pageSize, paginate }) => {
   return isCurrentPageValid ? store.get('currentPage') : revisedPage;
 };
 
+
 storiesOf('Table', module)
   .addParameters({
     info: {
       propTablesExclude: [State]
     }
   })
   .add('classic', () => {
-    const props = getCommonKnobs();
-    store.set({ currentPage: setCurrentPage(props) });
-    props.theme = select(
-      'theme',
-      [
-        OptionsHelper.tableThemes[0],
-        OptionsHelper.tableThemes[1]
-      ],
-      Table.defaultProps.theme
-    );
-
-    store.set({ sortOrder: props.sortOrder });
-    store.set({ sortedColumn: props.sortColumn });
-
-    return (
-      <ThemeProvider theme={ classic }>
-        <State
-          store={ store }
-          parseState={ state => ({ ...state, children: buildRows(props) }) }
-        >
-          <Table
-            actionToolbarChildren={ (context) => {
-              return [
-                <Button disabled={ context.disabled } key='single-action'>
-                    Test Action
-                </Button>,
-                <MultiActionButton
-                  text='Actions' disabled={ context.disabled }
-                  key='multi-actions'
-                >
-                  <Button>foo</Button>
-                  <Button>bar</Button>
-                  <Button>qux</Button>
-                </MultiActionButton>
-              ];
-            } }
-            path='/countries'
-            actions={ { delete: { icon: 'bin' }, settings: { icon: 'settings' } } }
-            { ...props }
-            onChange={ handleChange }
-            sortOrder={ store.sortOrder }
-            sortedColumn={ store.sortedColumn }
-          />
-        </State>
-      </ThemeProvider>
-    );
-  }, {
-    info: { text: info },
-    notes: { markdown: notes }
-  })
-  .add(
-    'default',
-    () => {
-      const props = getCommonKnobs();
-      store.set({ currentPage: setCurrentPage(props) });
-      props.size = select('size', OptionsHelper.tableSizes, Table.defaultProps.size);
-      props.isZebra = boolean('zebra striping', false);
-      props.theme = select(
+    const Wrapper = () => {
+      const [pageSize, setPageSize] = useState('10');
+
+      const handlePagination = (size) => {
+        setPageSize(size);
+      };
+
+      const tableProps = getCommonKnobs();
+      store.set({ currentPage: setCurrentPage(tableProps) });
+      tableProps.pageSize = tableProps.showPageSizeSelection ? pageSize : text('pageSize', '5');
```

Comment:
shouldn't that also be in `getCommonKnobs`?

Reply:
Agreed, I've separated the knobs out differently